### PR TITLE
Fix: minor time tense mistakes

### PR DESCRIPTION
**Pull request by 1634640**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] My article does not contain shortened URLs or affiliate links.

If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.

Closes #XXXXX

### CODE REVIEW

**Comment by 1634640** [comment]
Fixed minor mistakes in where words should be written in in future tense.

**Comment by RandellDawson** [comment]
Closed and Reopened this PR to attempt to resolve a specific Travis build failure.

File: guide/spanish/agile/safe/index.md [hunk 1]
```diff
@@ -12,13 +12,13 @@ El sitio web principal ( [http://www.scaledagileframework.com/](http://www.scale
 
 Hay nueve, principios subyacentes para SAFe
 
-1.  **Adopte una visión económica** : evalúe los costos económicos de decisiones, desarrollos y riesgos
+1.  **Adoptar una visión económica** : evalúar los costos económicos de decisiones, desarrollos y riesgos
```
Comment (hestellez) [inline guide/spanish/agile/safe/index.md]: "evaluar" no lleva tilde

File: guide/spanish/agile/safe/index.md [hunk 2]
```diff
@@ -12,13 +12,13 @@ El sitio web principal ( [http://www.scaledagileframework.com/](http://www.scale
...
 4.  **Construya de forma incremental con ciclos de aprendizaje rápidos e integrados** : las construcciones rápidas e incrementales permiten una retroalimentación rápida para cambiar el proyecto si es necesario
 5.  **Base de los hitos en la evaluación objetiva en sistemas de trabajo** : la evaluación objetiva proporciona información importante, como finanzas y gobernabilidad, como retroalimentación
 6.  **Visualice y limite la WIP, reduzca el tamaño de los lotes y administre las longitudes de cola** : haga esto para lograr un flujo continuo para moverse rápidamente y visualizar el progreso; WIP = trabajo en progreso
-7.  **Aplique cadencia, sincronice con la planificación de dominios cruzados** : la cadencia es el establecimiento de fechas en las que ocurren ciertos eventos (por ejemplo, el lanzamiento semanal) y la sincronización garantiza que todos tengan los mismos objetivos en mente
+7.  **Apliqar cadencia, sincronice con la planificación de dominios cruzados** : la cadencia es el establecimiento de fechas en las que ocurren ciertos eventos (por ejemplo, el lanzamiento semanal) y la sincronización garantiza que todos tengan los mismos objetivos en mente
```
Comment (hestellez) [inline guide/spanish/agile/safe/index.md]: Es "aplicar"

**Review by hestellez — CHANGES_REQUESTED** [review]

**Comment by RandellDawson** [comment]
Thank you for opening this pull request.

This is a standard message notifying you that we’ve reviewed your pull request and have decided not to merge it. We would welcome future pull requests from you.

Thank you and happy coding.

**Pull request closed without merging**